src/Arcus.EventGrid.Tests.Unit/Parsing/RawEventParsingTests.cs

```diff
@@ -75,7 +76,7 @@ public void ParseToRaw_ValidRawEvent_ShouldSucceed()
             Assert.Equal(originalEvent.EventTime, eventPayload.EventTime);
             Assert.Equal(originalEvent.DataVersion, eventPayload.DataVersion);
             Assert.NotNull(eventPayload.Data);
-            Assert.Equal(expectedEventPayload, eventPayload.Data);
+            Assert.Equal(rawEventBody, Regex.Replace(eventPayload.Data.ToString(), "\\s+", String.Empty));
```

Comment:
Ah, remove JSON indentation. I see that this can be a bit confusing. I'll change it by serializing with indentation so the assert becomes:

```csharp
Assert.Equal(rawEventBody, eventPayload.Data.ToString());
```

Reply:
Both the old approach (via `JToken`, but not ideal) or your suggestion are fine for me!